frontend/src/index.tsx

```diff
@@ -41,7 +41,11 @@ const PrivateRoute: React.FC<any> = ({ component: Component, ...rest }) => {
 const Routes = () => {
   return (
     <KeycloakProvider
-      authClient={keycloakClient(localStorage.getItem('keycloakAuthUrl'))}
+      authClient={keycloakClient(
```

Comment:
Does this mean running it locally with docker-compose won't work any more?

Reply:
This change is making it work locally with or without docker.